### PR TITLE
Cb edits

### PR DESCRIPTION
**Pull request by cboettig**
@mlap give this a quick look over and merge when ready?  Mostly just a few references, also a few small line-edits in intro/examples (and deleting stale comments)

### CODE REVIEW

File: manuscript/manuscript.Rmd [hunk 1]
```diff
@@ -124,11 +110,8 @@ Throughout this process, the agent's behavior is codified into what is called a
...
-<!-- ML: what are classical approaches below here? -->
-In contrast to classical approaches, there are few restrictions on what comprises a state or action. 
-The state may be continuous or discrete, completely or partially observed, single or multidimensional.
-For instance, we might represent an ecological community of 10 interacting species with a 10-dimensional continuously valued state space signifying the population density of each species.
-The action space can be similarly complex, since an action can be continuous or discrete, single or multidimensional.
+In contrast to classical approaches [@Marescot2013], there are few restrictions on what comprises a state or action. 
+States and actions may be continuous or discrete, completely or partially observed, single or multidimensional.
 For the previous environment, we might set the action space to be continuously valued and 10-dimensional, where each dimension corresponds to the amount of conservation effort aimed at the different species over a time step.
```
Comment (mlap) [inline manuscript/manuscript.Rmd:115]: will want to delete this line also

File: manuscript/manuscript.Rmd [hunk 2]
```diff
@@ -508,13 +473,12 @@ Rarely do we know the underlying functional forms for ecological processes -- is
...
 Model-free RL approaches offer a powerful way to factor this thorny issue.
-The model-free RL agent needs to make no such a priori hypothesis about a particular functional form. <!-- ML: I feel like the remaining part of this dont full drive home the benefit of model-free rl, like its able to learn good performing policies even in the case when a model is very difficult or impossible to learn. -->
-Alternative models can easily be implemented in simulations, allowing an agent to train against both contingencies.
-A successful agent will learn a policy that works well in both scenarios.
-This is analogous to classical approaches for integrating over parameter or model uncertainty [e.g. @Walters1981; @Ludwig1982],
+The model-free RL agent needs to make no such a priori hypothesis about a particular functional form.
+Model-free approaches have proven successful at learning effective policies even when the underlying model may be difficult or impossible to learn [@alphago], as long as simulations of possible mechanisms are available.
```
Comment (mlap) [inline manuscript/manuscript.Rmd:477]: In an edit that may be lost now on thelio 😬 , I tried to mention here that the concern with the model-free approach is sample complexity -- not a concern if you have a simulator but could be problematic if you are trying to learn directly from data via offline algorithms. 

AlphaGo is also probably not the best citation here -- some aspects I'd call model-based. [https://arxiv.org/pdf/1802.09081.pdf](https://arxiv.org/pdf/1802.09081.pdf) seems to at least hammer the comparison here although does not show it or give a solid citation for it.

File: manuscript/manuscript.Rmd [hunk 3]
```diff
@@ -508,13 +473,12 @@ Rarely do we know the underlying functional forms for ecological processes -- is
...
 
 <!-- Real problems are harder than these examples -->
 <!-- ML: I feel like this paragraph could be condensed with the one above -->
+<!-- CB: I think "model free is great" and "real problems are harder than these examples" are two distinct messages though and at least one would get lost in mashing them together.  Perhaps the message doesn't come across in the current version.  Does look like this paragraph is similar in overall message to the next one though, maybe could be combined with that?  -->
```
Comment (mlap) [inline manuscript/manuscript.Rmd:481]: Yeah I think this might have been a vocab mischoice by me with above vs. below.

**Review by mlap — APPROVED** [review]

**Comment by cboettig** [comment]
Thanks @mlap .  I'll swap out the citation and try and combine those two paragraphs (and the extraneous sentence on L115)

**Pull request merged**